src/main.js

```diff
@@ -59,15 +71,35 @@ const parseVarFromType = envSpecAsArray => {
 const outputHTML = envValues => {
   //create HTML format
   if (envValues) {
-    envValues = envValues.map(
-      element =>
-        `<label for="env_spec_${element[0].toLowerCase()}">${
-          element[0]
-        }</label>\n` +
-        `<input id="env_spec_${element[0].toLowerCase()}" name="${element[0].toLowerCase()}" type="${
-          element[1]
-        }" />\n`
-    );
+    envValues = envValues.map(element => {
+      switch (element[2]) {
+        case 0: //if element is value with type
+          toPrint =
+            `<label for="env_spec_${element[0].toLowerCase()}">${
+              element[0]
+            }</label>\n` +
+            `<input id="env_spec_${element[0].toLowerCase()}" name="${element[0].toLowerCase()}" type="${
+              element[1]
+            }" />\n`;
+          return toPrint;
+
+        case 1:
+          //if element is value with restricted choices
+          toPrint =
+            `<label for="env_spec_admin_${element[0].toLowerCase()}">${
```

Comment:
Since the code for the output label is the same whether we have types, restricted options or nothing, we should extract this to a function that we reuse in both places.